packages/mongo-nav/src/org-nav/OrgNav.tsx

```diff
@@ -109,6 +123,7 @@ export default function OrgNav({
   hosts,
 }: OrgNav) {
   const { orgNav } = urls;
+  const disabled = activeNav === 'preferences';
```

Comment:
I'm wondering if the name for this `activeNav` value should be more general than 'preferences'. It's possible we may need this for somewhere other than preference pages.

Reply:
I took this from Brooke, but if you provide a better name, I'll change it. I feel uncomfortable trying to rename this without a suggestion that you'd prefer.

Reply:
@DesignerDave do you prefer `userPreferences`? For now, all of the activeNav values have been matched to possible `activeNav` states

Reply:
My thought is more around perhaps not needing to check for `disabled` and `current` separately and finding a way to combine that logic, but that might be for another PR once I understand all of the moving parts that go into determining whether or not to show a payment badge... 🤔

Reply:
@hswolff `atlasPreferences` seems quite misleading to me. The page that sets this active nav is the preferences (for Atlas/CM) or account (for OM) page. Those pages do not deal with anything Atlas/CM/deployment/cluster/etc related.

Were this only for atlas/cloud manager, I'd suggest `userPreferences`. Were this only for Ops Manager I'd suggest `userAccount`. I think a good compromise would be `userSettings` instead.

Reply:
Yeah I guess the full name would be `atlasUserPreferences`.  `userSettings` wfm.

Reply:
I still wouldn't want `atlasUserPreferences`, because then if I'm an Ops Manager user updating my username, the page is (Ops Manager) Account Settings. `atlasUserPreferences` both isn't platform-agnostic and doesn't describe the page, due to our unfortunately bifurcation of preferences vs. account settings.

Is this just to separate things from the potential `realmUserSettings` and `chartsUserSettings`, or some global `userSettings`? If so, we can definitely change it to `cloudUserSettings`?

I also think this might be worth addressing in a pre-release review rather than during alpha, if there is still disagreement.